react-carousel-component/src/carousel.jsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+
+class Carousel extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      id: 1
+    };
```

Comment:
While using the `id` here works because you directly defined the ids in the array that won't always be the case. Normally we get data from databases and the databases auto generate ids and they are almost never in perfect order. So a better solution for this would to be use the items index to display it. The index of the array will always remain in order even if item are added or removed from the array. You DO NOT need to change this, just as an FYI for future endeavors.